index.js

```diff
@@ -52,19 +52,19 @@ var m = {
 	},
 	// success
 	sline: function(char, length) {
-		linesUtil.logLine(char, length, this.e, this.w, config.colors.success);
+		linesUtil.logLine(char, length, this.e, this.w, config.colors.logs.success);

```

Comment:
seems we are touching this piece of code anyways...might as well do the line config as well in this PR, no?